escape_roomba/thread_manager_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for thread_manager.ThreadManager."""
+
+import logging
+
+import discord
+import discord.abc
+import pytest
+
+from escape_roomba.context import Context
+from escape_roomba.thread_manager import ThreadManager
+
+
+@pytest.mark.asyncio
+async def test_thread_creation(mocker, make_context):
+    # Create a Context with a discord.Client-like Mock.
+    context = make_context(guild_count=2, channel_count=2, member_count=2)
+
+    # Create the ThreadManager and register its listeners.
+    ThreadManager(context)
+
+    # TODO: Finish this!
+    # await context.client.on_ready()
```

Comment:
Is this for a follow-up PR, or did you forget this? You could just leave this file out of this PR, since it doesn't do anything yet. 😺

Reply:
Follow-up PR (which is in progress). I like to check in stubs and TODO's even if they're works in progress (otherwise they live in an odd limbo state), do you object to that?

Reply:
No, I do not object.